### ISSUE TITLE
Block/unblock per app

### ISSUE THREAD

**Issue by colbycdev** [blokadaorg/blokada#310]
I use a game called Subway Surfers (yes, I still play it) that uses graph.facebook.com for leaderboard, so I have to whitelist it. However, I don't want it unblocked everywhere. I want a way to say "if app=Subway surf, then whitelist graph.facebook.com, else block graph.facebook.com".
I know Netguard does a similar thing (though not exactly). I guess what I really want is the best of blokada mixed with the best of netguard (which is open source).

**Comment by rpeter85** [comment]
Whitelist the application: https://go.blokada.org/help#whitelist

**Comment by fmilligan** [comment]
But then I see very intrusive ads, which I don't want.  I only  want to see the leaderboard

**Comment by rpeter85** [comment]
You will see the same intrusive ads if you whitelist the domain which serves ads.

**Comment by colbycdev** [comment]
No, the domain isn't where the ads are served from. They're served from ads.api.vungle.com, and touch.inmobi.com.
graph.facebook.com just serves data for the leaderboard.

We all work for someone to

________________________________

**Comment by rpeter85** [comment]
I see. So you want to block the tracker in every app but leak all the data in another.

**Comment by colbycdev** [comment]
A necessary evil. Look, if you have no interest in this issue, I can just close it and deal with having to whitelist the domain globally

**Comment by jeroen7s** [comment]
Also want this
I use a social blocking list to block all requests to facebook, except for a few which require it
Shouldn't be forced to unblock an entire app, you should be able to whitelist a specific host only for a specific app

**Comment by immertroll** [comment]
I also think this feature would be a big step to control the data. I would really like to use the Twitter app but without analytics and just there. The Twitter API should be blocked in any other app.
